jenkins/README.md

```diff
@@ -8,6 +10,18 @@
 * [Docker](./docker)
 * [Test](./test)
 
+
+### Check for Build
+
+To ensure that builds are trigger as changes come in the [Check-for-build](check-for-build.jenkinsfile) job is setup with a cron schedule to check manifests and run them at the appropriate build job.  From in a manifest a stable build version can be created and reference against uploaded versions.  If there is an exist match the build will be skipped, if not the job will be triggered in its own build workflow.
+
+For concurrent builds the Check-for-build job has an internal locks to prevent interruption for the same manifest and target job name, any requests that match an existing lock will be skipped to prevent a backup of queued build jobs
+
+#### Add jobs to run frequently
+
+* For jobs that are in active development they should be set to `H/10 * * * *` which will check every 10 minutes 
```

Comment:
Set cron to .... for jobs that are, otherwise it's not very English.

Missing periods.